eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/DbsnpSubmittedVariantEntity.java

```diff
@@ -23,6 +23,10 @@
         super();
     }
 
+    public DbsnpSubmittedVariantEntity(Long accession, String hashedMessage, ISubmittedVariant model) {
```

Comment:
If the constructor without version in `SubmittedVariantEntity` is not needed (check next comment first) then this would have to be deleted.

Reply:
Deleted